.github/workflows/ci.yml

```diff
@@ -54,12 +54,12 @@ jobs:
 
   # Run the C integration tests.
   c-tests:
-    uses: lf-lang/lingua-franca/.github/workflows/c-tests.yml@master
+    uses: lf-lang/lingua-franca/.github/workflows/c-tests.yml@auth
     needs: cancel
 
   # Run the CCpp integration tests.
   ccpp-tests:
-    uses: lf-lang/lingua-franca/.github/workflows/c-tests.yml@master
+    uses: lf-lang/lingua-franca/.github/workflows/c-tests.yml@auth
```

Comment:
Changed `install-rti/install.sh` and `c-tests.yml` so changed this temporarily. Need to change back to master after merged.

Reply:
The way to do  it is to keep it like this; merge (not yet remove the branch!); and push a change to `master` to update these lines. After that, the branch can be safely deleted. I usually take care of this, so you don't have to worry about it.